### PR TITLE
Arc 2376 ner datamodel

### PR DESCRIPTION
**Pull request by lennertvandevelde**

### CODE REVIEW

File: ontologies/ebucore.rdfs.ttl [hunk 1]
```diff
@@ -163,3 +174,47 @@ ebucore:hasCastMember a rdf:Property ;
   rdfs:comment "A member of the cast."@en, "Een lid van de cast."@nl, "A member of the cast."@fr, "Range: a string or a Cast"@en ;
   rdfs:range xsd:string, ebucore:Cast ;
   rdfs:domain ebucore:Asset .
+
+ebucore:regionDelimX a rdf:Property ;
+  rdfs:domain ebucore:MediaFragment ;
+  rdfs:range xsd:string, xsd:anyURI ;
+  rdfs:subPropertyOf ebucore:hasAnnotationPurpose ;
+  rdfs:label "Region delimiter ()"@en ;
```
Comment (mielvds) [inline ontologies/ebucore.rdfs.ttl:182]: Something missing here?

File: ontologies/ebucore.rdfs.ttl [hunk 2]
```diff
@@ -163,3 +174,47 @@ ebucore:hasCastMember a rdf:Property ;
...
+  rdfs:label "Délimiteur de région (axe des y)"@nl ;
+  skos:definition "To define the top left corner of a zone on the y-axis."@en ;
+  skos:definition "Définir le coin supérieur gauche d'une zone sur l'axe des y."@fr ;
+  skos:definition "Het definiëren van de linkerbovenhoek van een zone op de y-as."@nl .
+
+ebucore:width a rdf:Property ;
+  rdfs:domain ebucore:MediaFragment ;
+  rdfs:range xsd:string, xsd:anyURI ;
+  rdfs:subPropertyOf ebucore:hasAnnotationPurpose ;
+  rdfs:label "Width"@en ;
```
Comment (mielvds) [inline ontologies/ebucore.rdfs.ttl:204]: lowercase? (or are other ebucore properties also capitalized?)

File: organizations/organizations.shacl.ttl [hunk 3]
```diff
@@ -768,6 +768,20 @@
     sh:description "Le prénom et le nom de cette personne (combinés en une seule string)."@fr ;
     sh:description "De voor- en achternaam van deze persoon (gecombineerd in één string)."@nl ;
 
+    sh:severity sh:Violation ;
+    sh:message "occurs more than once or its object is not of type xsd:string"@en ;
+  ],
+  [
+    a sh:PropertyShape ;
+    sh:path schema:deathPlace ;
```
Comment (mielvds) [inline organizations/organizations.shacl.ttl:776]: I don't immediately spot it, but `schema:birthPlace` is added as well?

File: ontologies/ebucore.rdfs.ttl [hunk 4]
```diff
@@ -145,6 +145,17 @@ ebucore:hasAnnotationRelatedArtefact a rdf:Property ;
   skos:definition "Pour identifier un objet d'artefact de l'annotation."@fr ;
   skos:definition "Om een artefactonderwerp van de annotatie te identificeren."@nl .
 
+ebucore:hasAnnotationRelatedAgent a rdf:Property ;
```
Comment (mielvds) [inline ontologies/ebucore.rdfs.ttl:148]: FWIW, you can remove [ebucore:hasAnnotationPurpose](http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#hasAnnotationPurpose), [ebucore:hasAnnotationRelatedAgent](https://www.ebu.ch/metadata/ontologies/ebucore/ebucore_hasAnnotationRelatedAgent.html), and [ebucore:hasAnnotationRelatedLocation](https://www.ebu.ch/metadata/ontologies/ebucore/ebucore_hasAnnotationRelatedLocation.html). They can all be replaced by `ebucore:hasAnnotationAgent`

File: ontologies/ebucore.rdfs.ttl (hunk 2 above)
Comment (lennertvandevelde) [reply ontologies/ebucore.rdfs.ttl:204]: It is a mix of both, so I will all put them in lowercase

File: ontologies/ebucore.rdfs.ttl (hunk 4 above)
Comment (lennertvandevelde) [reply ontologies/ebucore.rdfs.ttl:148]: Maybe we should comment them out for now? We do not have any instances yet, but with the future NER data we will probably need them again.

**Pull request merged**